src/Microsoft.Health.Fhir.TemplateManagement/Utilities/StreamUtility.cs

```diff
@@ -8,61 +8,83 @@
 using System.IO;
 using System.Linq;
 using System.Security.Cryptography;
-using System.Text;
-using ICSharpCode.SharpZipLib.GZip;
-using ICSharpCode.SharpZipLib.Tar;
+using EnsureThat;
 using Microsoft.Health.Fhir.TemplateManagement.Exceptions;
 using Microsoft.Health.Fhir.TemplateManagement.Models;
+using SharpCompress.Common;
+using SharpCompress.Readers;
+using SharpCompress.Writers;
+using SharpCompress.Writers.Tar;
 
 namespace Microsoft.Health.Fhir.TemplateManagement.Utilities
 {
     public class StreamUtility
     {
-        public static Dictionary<string, byte[]> DecompressTarGzStream(Stream sourceStream)
+        private static DateTime GzipTimestamp => new DateTime(2021, 1, 1);
```

Comment:
It would be better if we can add some comments here or add some information in the document for this constant.

Reply:
Can we use `new DateTime(1970, 1, 1)` directly here?

Reply:
Thanks, refine it and reset timestamp to 0.